classes/task/fetch_recording_metadata_task.php

```diff
@@ -193,17 +193,34 @@ public function execute() {
 
                 // Mark the record if no recording was detected, or should be considered recording-free.
                 if (!isset($recordings[$recordid])) {
+                    // Set a lastchecked timestamp, to ensure subsequent requests within a
+                    // certain threshold (defaulting to 20 minutes) are not made and the
+                    // requesting is skipped.
+                    $meta->lastchecked = time();
+
                     // Check and see if the record has a lastchecked already linked to it.
                     // If it has surpassed the threshold, it is safe to consider the meeting
                     // as having not been recorded.
-                    if (time() > $meeting->timecreated + TIME_BEFORE_FLAGGING_AS_NOT_RECORDED) {
+                    if (time() > ($meeting->timecreated + TIME_BEFORE_FLAGGING_AS_NOT_RECORDED)) {
```

Comment:
This I believe is actually a bug fix. Stepping through the debugger, the time check here seems to always apply it?? It might be due to it evaluating the LHS, which makes it zero, then adding some fixed number making it truthy.

Either way, wrapping this in parenthesis should remove this concern.